controllers/idler/idler_controller.go

```diff
@@ -197,10 +212,18 @@ func (r *Reconciler) createNotification(logger logr.Logger, idler *toolchainv1al
 			// no email found, thus no email sent
 			return fmt.Errorf("no email found for the user in MURs")
 		}
-
+		if len(appName) == 0 {
+			appName = ""
+		}
+		if len(appType) == 0 {
+			appType = ""
+		}
 		keysAndVals := map[string]string{
 			"Namespace": idler.Name,
+			"Appname":   appName,
+			"Apptype":   appType,
```

Comment:
out of curiosity - is there any reason for not using the CamelCase 
```suggestion
			"AppName":   appName,
			"AppType":   appType,
```

Reply:
No Specific reason.. I shall update it to camel case

Reply:
+1 to CamelCase.